core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala

```diff
@@ -104,6 +114,51 @@ object SnappyTableStatsProviderService extends Logging {
       delay, delay)
   }
 
+
+  def getAggregatedMemberStatsOnDemand: Unit = {
```

Comment:
This method should be inside a try.catch block. So that any exception won't show up.